pkg/backup/backup_test.go

```diff
@@ -26,6 +26,28 @@ import (
 	"github.com/coreos/etcd-operator/pkg/util"
 )
 
+func TestResponseHeaderIsCorrect(t *testing.T) {
+	d, err := setupBackupDir("3.1.0_000000000000000a_etcd.backup")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(d)
+	b := &Backup{
+		be: &fileBackend{dir: d},
+	}
+	req := &http.Request{
+		URL: util.MakeBackupURL("ignore", ""),
+	}
+	rr := httptest.NewRecorder()
+	b.serveSnap(rr, req)
+	if get := rr.Header().Get(HTTPHeaderEtcdVersion); get != "v3.1.0" {
+		t.Fatalf("etcd version want=%s, get=%s", "v3.1.0", get)
```

Comment:
use t.Errorf for test case error. (fatal should be used for the setup phase only)